### PR TITLE
Allow user to select on which SR to run quicktest

### PR DESCRIPTION
**Pull request by gthvn1**

### CODE REVIEW

**Comment by gthvn1** [comment]
Koji scratch build: https://koji.xcp-ng.org//taskinfo?taskID=36928

File: SOURCES/xapi-1.249.32-allow-a-user-to-select-on-which-SR-to-run-quicktest.backport.patch [hunk 1]
```diff
@@ -0,0 +1,95 @@
+From 5a07265ef43a3d50b586ca029759ca0638705bcb Mon Sep 17 00:00:00 2001
+From: BenjiReis <benjamin.reis@vates.fr>
+Date: Tue, 18 Jul 2023 13:24:18 +0200
+Subject: [PATCH] Allow a user to select on which SR to run quicktest
```
Comment (stormi) [inline SOURCES/xapi-1.249.32-allow-a-user-to-select-on-which-SR-to-run-quicktest.backport.patch:4]: If you had to adapt the patch for it to apply, you should add a comment on top of this, explaining your changes.

**Review by benjamreis — APPROVED** [review]

**Review by stormi — APPROVED** [review]

**Comment by gthvn1** [comment]
Please don't build for the moment we are waiting for another backport related to quicktest.

**Comment by gthvn1** [comment]
Build with #47

**Pull request merged**